src/app/3d/qgs3ddebugwidget.h

```diff
@@ -0,0 +1,41 @@
+/***************************************************************************
+  qgs3ddebugwidget.h
+  --------------------------------------
+  Date                 : November 2024
+  Copyright            : (C) 2024 by Matej Bagar
+  Email                : matej dot bagar at lutraconsulting dot co dot uk
+ ***************************************************************************
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License as published by  *
+ *   the Free Software Foundation; either version 2 of the License, or     *
+ *   (at your option) any later version.                                   *
+ *                                                                         *
+ ***************************************************************************/
+
+#ifndef QGS3DDEBUGWIDGET_H
+#define QGS3DDEBUGWIDGET_H
+
+class QStandardItemModel;
+class Qgs3DMapCanvas;
+
+#include <ui_3ddebugwidget.h>
+#include "qgs3dmapsettings.h"
```

Comment:
we don't need to include qgs3dmapsettings.h here, only in the cpp - here we can have just forward declaration

Reply:
compiler doesn't agree on this :( so it will have to stay